test/compound/TestSupplyFuzz.t.sol

```diff
@@ -0,0 +1,574 @@
+        amount = bound(amount, 1, DAI_BALANCE);
+        // vm.assume(MIN_ETHER_AMOUNT <= amount && amount <= MAX_ETHER_AMOUNT);
+        // uint256 amount = 10_000 ether;
+
+        vaultSupplier1.depositVault(daiSupplyVault, amount);
+
+        (uint256 balanceInP2P, uint256 balanceOnPool) = morpho.supplyBalanceInOf(
+            cDai,
+            address(daiSupplyVault)
+        );
+
+        uint256 p2pSupplyIndex = morpho.p2pSupplyIndex(cDai);
+        uint256 poolSupplyIndex = ICToken(cDai).exchangeRateCurrent();
+
+        assertGt(daiSupplyVault.balanceOf(address(vaultSupplier1)), 0, "mcDAI balance is zero");
+        assertApproxEqAbs(
+            balanceInP2P.mul(p2pSupplyIndex) + balanceOnPool.mul(poolSupplyIndex),
+            amount.div(poolSupplyIndex).mul(poolSupplyIndex),
+            1e10
+        );
+    }
+
+    function testShouldWithdrawAllAmountFuzz(uint256 amount) public {
+        amount = bound(amount, 1, DAI_BALANCE); // @bug падает по точности, падает при amount == 1
+
+        uint256 poolSupplyIndex = ICToken(cDai).exchangeRateCurrent();
+        uint256 expectedOnPool = amount.div(poolSupplyIndex);
+
+        vaultSupplier1.depositVault(daiSupplyVault, amount);
+        vaultSupplier1.withdrawVault(daiSupplyVault, expectedOnPool.mul(poolSupplyIndex));
+
+        (uint256 balanceInP2P, uint256 balanceOnPool) = morpho.supplyBalanceInOf(
+            cDai,
+            address(daiSupplyVault)
+        );
+
+        assertApproxEqAbs(
+            daiSupplyVault.balanceOf(address(vaultSupplier1)),
+            0,
+            3e3,
+            "mcDAI balance not zero"
+        );
+        assertEq(balanceOnPool, 0, "onPool amount not zero");
+        assertEq(balanceInP2P, 0, "inP2P amount not zero");
+    }
+
+    function testShouldWithdrawAllUsdcAmountFuzz(uint256 amount) public {
+        amount = bound(amount, 1e3, 1e12); // @bug падает при 1 - депозит проходит, но withdraw падает
+        // uint256 amount = 1e9;
+
+        uint256 poolSupplyIndex = ICToken(cUsdc).exchangeRateCurrent();
+
+        uint256 expectedOnPool = amount.div(poolSupplyIndex);
+
+        vaultSupplier1.depositVault(usdcSupplyVault, amount);
+        vaultSupplier1.withdrawVault(usdcSupplyVault, expectedOnPool.mul(poolSupplyIndex));
+
+        (uint256 balanceInP2P, uint256 balanceOnPool) = morpho.supplyBalanceInOf(
+            cUsdc,
+            address(usdcSupplyVault)
+        );
+
+        console.logUint(usdcSupplyVault.balanceOf(address(vaultSupplier1)));
+        assertApproxEqAbs(
+            usdcSupplyVault.balanceOf(address(vaultSupplier1)),
+            0,
+            10,
+            "mcUSDC balance not zero"
+        );
+        assertEq(balanceOnPool, 0, "onPool amount not zero");
+        assertEq(balanceInP2P, 0, "inP2P amount not zero");
+    }
+
+    function testShouldWithdrawAllSharesFuzz(uint256 amount) public {
+        amount = bound(amount, 1, DAI_BALANCE); // @bug падает при маленьких amount
```

Comment:
Why `args` 0?
```
[FAIL. Reason: Assertion failed. Counterexample: calldata=0x18a504c10000000000000000000000000000000000000000000000000000000000000000, args=[0]] testShouldWithdrawAllSharesFuzz(uint256) (runs: 1, μ: 952576, ~: 952576)
Logs:
  Bound Result: 1
```

Reply:
It is bounded to `1` with `bound`: `Bound Result: 1`

Reply:
Got it.